bower.json

```diff
@@ -33,11 +33,9 @@
     "README.md"
   ],
   "dependencies": {
-    "bootstrap": "~3.3.6",
     "angular": "^1.5.0",
     "angular-sanitize": "^1.5.0",
-    "angular-bootstrap": "^1.1.2",
-    "angular-translate": "^2.9.2"
```

Comment:
Any reason this was removed?

Reply:
It's been a very long time since I opened this PR but I suspect that `angular-translate` is not required by the `angular-dialog-service`

Reply:
Looking at your repo again I see that `angular-translate` is definitely required. I'm not sure why I removed it.

Reply:
Taking a closer look I see that `angluar-translate` is in fact an optional dependency. As such, it should not be marked as a bower dependency.

Reply:
Thanks!